Mathlib/Probability/Distributions/Uniform.lean

```diff
@@ -172,9 +185,33 @@ theorem integral_eq (huX : IsUniform X s ℙ) :
 
 end IsUniform
 
-end pdf
+variable {X : Ω → E}
+
+lemma IsUniform.cond {s : Set E} :
+    IsUniform (id : E → E) s (ProbabilityTheory.cond μ s) μ := by
+  unfold IsUniform
+  rw [Measure.map_id]
+
+/-- The density of the uniform measure on a set with respect to itself. This allows us to abstract
+away the choice of random variable and probability space. -/
+def uniformPDF (μ : Measure E := by volume_tac) (s : Set E) (x : E) : ℝ≥0∞ :=
```

Comment:
I am not sure that works with volume_tac. If you write `uniformPDF s x` (without the measure), it might try to identify `s` with a measure. Can you test it, and put the measure at the end if it fails?

Reply:
In defining `IsUniform` we are also doing ` by volume_tac`, that's why I put it here. Writing `uniformPDF s x` fails. Should I just remove `:= by volume_tac`?

Reply:
You should put the measure as last argument to make it work.

Reply:
volume_tac doesn't seem to work, I've moved `μ` to the end.

Reply:
Why did you remove volume_tac? What I meant is that moving the measure to the end would allow volume_tac to work. This should beperfectly fine: `def uniformPDF (s : Set E) (x : E) (μ : Measure E := by volume_tac)`

Also, if you remove volume_tac (which I think you should not do), there is no reason to move the measure to the end.

Reply:
Okay, I totally misinterpreted what you said, sorry about that. I will put the volume_tac back!